cilium/cmd/monitor.go

```diff
@@ -175,12 +175,35 @@ func getMonitorParser(conn net.Conn, version listener.Version) (parser eventPars
 	}
 }
 
+func endpointExists(endpoint format.Uint16Flags) bool {
+	eps, err := client.EndpointList()
+	if err != nil {
+		Fatalf("cannot get endpoint list: %s\n", err)
+	}
+	for _, ep := range eps {
+		if endpoint.Has(uint16(ep.ID)) {
```

Comment:
It's not entirely clear here, but this returns true when the first endpoint is observed, so if there are several endpoints provided, only one of which is actually present, then monitor will operate without warning/fatal.

I think that this is close to the behaviour I'd expect, ie if at least one of the endpoints I want to monitor is present, then I'd want monitor to continue; However, it'd also be nice if it printed a warning to stderr stating that the other endpoints can't be found, to inform the user that the monitor log won't contain events from those endpoints.

Reply:
@joestringer this completely make sense but before I wasn't sure if it is possible to pass multiple endpoints I was trying `cilium monitor --to=91,919` :man_facepalming: